tensorflow/python/ops/parallel_for/pfor.py

```diff
@@ -4659,7 +4667,7 @@ def _init_values(self):
           # vectorization" format, so we want to keep it that way as much as
           # possible. We'll accumulate finished iterations (only relevant for
           # pfor-loop-variant while_loop conditions) in an accumulator with
-          # type:
+          # type : 
```

Comment:
nit: unnecessary change.

Reply:
It is required for https://github.com/python/typed_ast/issues/62. I don't know if it was released in a more recent `pylint`release.

Reply:
I've checked and it is not fixed in any release:
https://github.com/PyCQA/pylint/issues/4028#issuecomment-761575189